File: Difficulty: Easy/Pattern 12/pattern-12.java
class Solution {

    void printTriangle(int n) {
        // code here
        for(int i=1;i<=n;i++){
            for(int j=1;j<=i;j++){
                System.out.print(j+" ");
            }
            for(int space=1;space<=2*(n-i);space++){
                System.out.print("  ");
            }
            for(int j=i;j>=1;j--){
                System.out.print(j+" ");
            }
        System.out.println();
        }
    }
}

File: Difficulty: Easy/Pattern 12/pattern-12.py
#User function Template for python3

class Solution:
    def printTriangle(self, N):
        # Code here
        for i in range(1,N+1):
            for j in range(i):
                print(j+1,end=" ")
            for space in range(2*(N-i)):
                print("  ",end="")
            for j in range(i,0,-1):
                print(j,end=" ")
            print()